apm-sniffer/config/agent.config

```diff
@@ -15,100 +15,243 @@
+plugin.feign.collect_request_body=${SW_PLUGIN_FEIGN_COLLECT_REQUEST_BODY:false}
+#  When `COLLECT_REQUEST_BODY` is enabled, how many characters to keep and send to the OAP backend, use negative values to keep and send the complete body.  
+plugin.feign.filter_length_limit=${SW_PLUGIN_FEIGN_FILTER_LENGTH_LIMIT:1024}
+#  When `COLLECT_REQUEST_BODY` is enabled and content-type start with SUPPORTED_CONTENT_TYPES_PREFIX, collect the body of the request , multiple paths should be separated by `,`  
+plugin.feign.supported_content_types_prefix=${SW_PLUGIN_FEIGN_SUPPORTED_CONTENT_TYPES_PREFIX:application/json,text/}
+# If true, trace all the influxql(query and write) in InfluxDB access, default is true.
+plugin.influxdb.trace_influxql=${SW_PLUGIN_INFLUXDB_TRACE_INFLUXQL:true}
+#  Apache Dubbo consumer collect `arguments` in RPC call, use `Object#toString` to collect `arguments`. 
+plugin.dubbo.collect_consumer_arguments=${SW_PLUGIN_DUBBO_COLLECT_CONSUMER_ARGUMENTS:false}
+#  When `plugin.dubbo.collect_consumer_arguments` is `true`, Arguments of length from the front will to the OAP backend 
+plugin.dubbo.consumer_arguments_length_threshold=${SW_PLUGIN_DUBBO_CONSUMER_ARGUMENTS_LENGTH_THRESHOLD:256}
+#  Apache Dubbo provider collect `arguments` in RPC call, use `Object#toString` to collect `arguments`. 
+plugin.dubbo.collect_provider_arguments=${SW_PLUGIN_DUBBO_COLLECT_PROVIDER_ARGUMENTS:false}
+#  When `plugin.dubbo.provider_consumer_arguments` is `true`, Arguments of length from the front will to the OAP backend 
+plugin.dubbo.consumer_provider_length_threshold=${SW_PLUGIN_DUBBO_CONSUMER_PROVIDER_LENGTH_THRESHOLD:256}
+#  A list of host/port pairs to use for establishing the initial connection to the Kafka cluster. 
+plugin.kafka.bootstrap_servers=${SW_KAFKA_BOOTSTRAP_SERVERS:localhost:9092}
+#  Timeout period of reading topics from the Kafka server, the unit is second. 
+plugin.kafka.get_topic_timeout=${SW_GET_TOPIC_TIMEOUT:10}
+#  Kafka producer configuration.
+plugin.kafka.consumer_config=${SW_PLUGIN_KAFKA_CONSUMER_CONFIG:}
+# Kafka producer configuration. Read [producer configure](http://kafka.apache.org/24/documentation.html#producerconfigs)
+# to get more details. Check [Kafka report doc](How-to-enable-kafka-reporter.md) for more details and examples.
+plugin.kafka.producer_config=${SW_PLUGIN_KAFKA_PRODUCER_CONFIG:}
+#  Specify which Kafka topic name for Meter System data to report to. 
+plugin.kafka.topic_meter=${SW_PLUGIN_KAFKA_TOPIC_METER:skywalking_meters}
+#  Specify which Kafka topic name for JVM metrics data to report to. 
+plugin.kafka.topic_metrics=${SW_PLUGIN_KAFKA_TOPIC_METRICS:skywalking_metrics}
+#  Specify which Kafka topic name for traces data to report to. 
+plugin.kafka.topic_segment=${SW_PLUGIN_KAFKA_TOPIC_SEGMENT:skywalking_segments}
+#  Specify which Kafka topic name for Thread Profiling snapshot to report to. 
+plugin.kafka.topic_profilings=${SW_PLUGIN_KAFKA_TOPIC_PROFILINGS:skywalking_profilings}
```

Comment:
the property key should be `plugin.kafka.topic_profiling` and default value is `skywalking-profilings`